builtin/providers/vsphere/resource_vsphere_virtual_disk_test.go

```diff
@@ -0,0 +1,125 @@
+package vsphere
```

Comment:
@stack72 @jen20 @phinze say it ain't so ... no negative testing??  I will file an issues